### ISSUE TITLE
[Hunter][Spell] Piercing Shots - Scaling issues

### ISSUE THREAD

**Issue by hrtyv** [dalaranwow/dalaran-wow#4902]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)

Hunters talent "**Piercing Shots**" are not doing as much damage as it should. 
I have compared older (_2009-2011_) numbers from "World of logs" in Icecrown Citadel to the numbers i do on my Hunter and Hunters in same raid as myself on World of logs aswell. 
On average Piercing Shots do roughly 4-9% of your total damage on a boss fight on Dalaran-wow.

**Expected behaviour**: Tell us what should happen instead

This number was roughly 13-16% of your total damage on a boss fight in retail Wrath servers.
I am not 100% sure what causes the lower % damage but i can imagine some multipliers does not have the right values.

First i imagined that it was related to the lack of Armor penetration(ArP) rating in raids like ToC and Ulduar, so that the problem would solve itself in ICC with 1400 or 100% Armor penetration, but the damage % are still way too low.

- In the end it might be  Armor penetration that scales wrong on Hunters since Aimed shot and Steady 
   shot which aint changing much from 70% ArP to 100%, though there should be a big difference.

- Aimed Shot - Gains on average 1000 -1500 damage per shot with 100% Arp.
- Steady Shot - Gains on average 600 - 1200 damage per shot with 100% Arp.

- When Kill Shot goes from 28000 to 44000 damage on average per crit, with 100% Arp

The only shot that seems to scale well with armor pen on Dalaran-wow is **Killshot**. 
(_Which i can find proof of as well if needed_)
- Auto shot should also gain more from Armor penetration but i have not done enough research on that 
   to claim too much.

**Steps to reproduce the problem**:

1. Attack any enemy.
2. Critical strike hit with shots to proc Piercing Shots.
3. Notice numbers in % of damage done to enemies - Compare with retail values.

**Include proofs for this behaviour**

I have just picked a random fight in ICC25HC from my own recordings and found another random from retail servers, there are countless of other logs that shows Piercing Shots should do way more damage than it currently does.

My own records from WoL(Dalaran-WoW Values):
2017
https://i.imgur.com/CRegunF.jpg

Older records from WoL(Retail Values):
2010
https://i.imgur.com/1C8DTdk.jpg

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

Piercing shots
spell=63468

WoL source:
http://www.worldoflogs.com/

**Include Screenshots from the issue if necessary**

Screenshots above for proof

Hope this can raise some attention to this issue, ive looked through the forums for similar issues to not duplicate but couldn't find anything for this spell so here it is.

_I know Serpent Sting Chimera does more damage than it should as well but that does still not compensate for the loss in damage, because pets are doing way lower damage than they should too._ 

Thanks for reading

**Comment by TeridanDW** [comment]
Hello @hrtyv, thank you for the detailed report.
I have checked how Piercing shots works on ptr and after few basic tests it seems to work properly, meaning on each rank it does the proper %damage as bleed shown in the talent description. Here are some screenshots for each rank:
![wowscrnshot_101517_123631](https://user-images.githubusercontent.com/6797611/31583495-ee7370d8-b1a5-11e7-824f-6a33b98bd41e.jpg)
![wowscrnshot_101517_123151](https://user-images.githubusercontent.com/6797611/31583496-ee95d61e-b1a5-11e7-856f-6c5cd1444c52.jpg)
![wowscrnshot_101517_122622](https://user-images.githubusercontent.com/6797611/31583497-eeb3212e-b1a5-11e7-8f7e-68154fd70cc1.jpg)

However after reading your report 2 more times it seems that it is more directed towards how armor penetration works for hunters in general than Piercing shots.

Note: Keep in mind that directly checking logs on dalaran vs retail (wotlk) might not be very accurate source in this case because it matters how long was each fight, how many Aimed, Steady, Chimera shot you were able to use and how many of them were actually critical strikes.

Regarding how armor penetration works, we need more facts in order to say there is something wrong. I will still put the labels, but I'm not sure there is an issue here.

**Comment by hrtyv** [comment]
I get your point.

I also realized that doing this post it might be armorpen thats the "bad guy" at the end of this.
(also mentioned shortly in the post above).

The skill in itself are working just fine, but the scaling from armorpen does not seem to be correct implemented in the damage equation for the skill.

I made this post in hopes of getting some attention towards the Armorpen scaling on Hunter shots & Piercing Shots as the main topic. 
(Because i noticed the biggest % diference in that specific skill).

The thing is, i had some trouble finding the right equations for Armorpenetration rating 
scaling to actually come up with a decent post for it. 
So i made this since i suspected Piercing Shots to be bad scaling with armorpen together with Auto Shot - Steady Shot & Aimed Shot. 
This thing about Piercing Shots is that it should be a bigger part of Hunters dps and not having this scale as it should, makes the class "Not working as intended" in some way and basicly makes Armorpen "useless" compared to Agility stacking.

One thing that i think its the strangest thing about this is that Kill Shot seem to scale just fine when every other shot are not. 
(Can provide proof of this if needed)

Again thanks for reading and i hope this gonna raise some attention towards this issue.

**Comment by TeridanDW** [comment]
Regarding this:

>  Armorpen "useless" compared to Agility stacking

is there any chance the logs from retail were actually from a hunter who stacked AGI?
Anyway all we do here is speculate without any decisive proof. If anyone can provide one please do so below, until then putting on hold.

**Comment by STdimarx** [comment]
isnt this talent a bleed effect? If so, arp should affect it at all, right?

**Comment by Undertower** [comment]
ArP does not affect this talent. It does exactly 30% of the damage of Aimed, Chimera, and Steady Shot regardless of your ArP.
ArP increases the damage of Steady and Aimed, increasing the damage of Piercing Shots in the process.
Piercing Shots is a bleed and is affected by Mangle.

If you add the Critical Strike Damage of Chimera, Steady and Aimed from OP together it's:
396044(Chimera) + 410144(Steady) + 269740(Aimed) * 0.3(Piercing Shots) = 322778 damage
His Piercing Shots did 314104 damage which is working as intended.

Op was also missing 30% bleed damage from Mangle and using Arcane Shot which doesn't trigger Piercing Shots, further reducing the damage.

On retail Piercing Shot was affected by %damage so it did more than 30% of the damage you expect, however here on dalaran-wow it does exactly 30% no matter what.

Tl;dr: Piercing Shots works. Stop using Arcane Shot and make sure you bring a feral druid to maximize the damage of it.
